.rtd-environment.yml

```diff
@@ -0,0 +1,7 @@
+name: rtd_ebtelplusplus
+channels:
+  - conda-forge
+dependencies:
```

Comment:
Does the documentation compile and run any of the code?

Reply:
Yes there are several examples that are run in the gallery.

Reply:
Does that not need boost to compile or is it an optional extension?

Reply:
Boost gets installed on RTD via apt. That's specified in the `.readthedocs.yml` file.

Reply:
Oh I just somehow missed that. Sorry for the noise.